src/rest-server/src/config/v2/protocol.js

```diff
@@ -317,6 +317,14 @@ const protocolSchema = {
         },
       },
     },
+    extras: {
+      type: 'object',
+      properties: {
```

Comment:
`extras` is an uncontrolled and free for adding any sub fields by any plugins, so it could not be limited with only `submitForm` here.

Refer: https://github.com/Microsoft/pai/blob/f52d98ec34da4d6d192f5f715b391e60d0b8adb3/src/rest-server/src/config/job.js#L108-L112

Reply:
removed properties